pkg/api/security/security.go

```diff
@@ -244,7 +244,7 @@ func saveAuthToken(token, tokenPath string) error {
 		return err
 	}
 
-	if err := perms.RestrictAccessToUser(tokenPath); err != nil {
+	if err := perms.RestrictAccessToUser(tokenPath); err == nil {
```

Comment:
This change is good, but wouldn't be more idiomatic to actually swap the bodies of the if and else branches ? I think we rarely see `err == nil` in any code.

Reply:
Otherwise thank you for the investigation and for this PR

Reply:
Good point, let me do that :)

Reply:
@paulcacheux fixed